packages/kbn-es/src/cluster.js

```diff
@@ -246,7 +246,7 @@ exports.Cluster = class Cluster {
     this._log.info(chalk.bold('Starting'));
     this._log.indent(4);
 
-    const esArgs = ['indices.query.bool.max_nested_depth=100'].concat(options.esArgs || []);
```

Comment:
changing the max depth back from 100 to 10.  This was set to 100 when elasticsearch started enforcing the default depth of 20 in https://github.com/elastic/kibana/commit/e914a60ba8c335a187e8a269596e5072ed4502ee

Reply:
I like having a lower max_nested_depth as an early warning sign that things are growing, but I also imagine the first time someone hits that limit because they reached a depth of 11 it will be pretty confusing... We should probably handle that error well but, idk, I'd recommend reverting this change.